File: Examples/PHP/managing-recognition/cloud-storage/recognize-specified-count-of-barcodes.php
<?php
require_once realpath(__DIR__ . '/../..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/../..') . '/Utils.php';

use Aspose\Barcode\BarcodeApi;
use Aspose\Barcode\AsposeApp;

class RecognizeBarcode {

	public $barcodeApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->barcodeApi = new BarcodeApi();
	}

	public function recognizeSpecifiedCountOfBarcodes() {

		// Set input file name
		$name = "sample-barcode.jpeg";

		// The barcode type.
		// If this parameter is empty, autodetection of all supported types is used.
		$type = "";

		// Set mode for checksum validation during recognition
		$checksumValidation = "On";

		// Set if FNC symbol stripping should be performed.
		$stripFnc = "True";

		// Set recognition of rotated barcode
		$rotationAngle = "";

		// Set exact number of barcodes to recognize
		$barcodesCount = "1";

		// Set recognition of barcode inside specified Rectangle region
		$rectX = 0;
		$rectY = 0;
		$rectWidth = 0;
		$rectHeight = 0;

		// Set 3rd party cloud storage server (if any)
		$storage = "";

		// Set folder location at cloud storage
		$folder = "";

		// Set local file (if any)
		$file = null;

		// upload file to aspose cloud storage
		Utils::uploadFile($name);

		// invoke Aspose.BarCode Cloud SDK API to read barcode
		$response = $this->barcodeApi->GetBarcodeRecognize ( $name, $type, $checksumValidation, $stripFnc, $rotationAngle, $barcodesCount, $rectX, $rectY, $rectWidth, $rectHeight, $storage, $folder );
		print_r ( $response );
	}
}

$recognizeBarcode = new RecognizeBarcode();
$recognizeBarcode->recognizeSpecifiedCountOfBarcodes();
?>

File: Examples/Android/app/src/main/java/barcode/cloud/aspose/com/barcodeexample/managingRecognition/cloudStorage/RecognizeSpecifiedcountOfBarcodes.java
package barcode.cloud.aspose.com.barcodeexample.managingRecognition.cloudStorage;

import android.util.Log;

import java.io.File;
import java.util.List;

import com.aspose.barcode.api.BarcodeApi;
import com.aspose.barcode.model.Barcode;
import com.aspose.barcode.model.BarcodeResponseList;
import com.aspose.storage.api.StorageApi;

import barcode.cloud.aspose.com.barcodeexample.utils.Configuration;

/**
 * This sample code allows you to recognize specified count of barcodes on the
 * Aspose Cloud Storage using Aspose BarCode Cloud API.
 * 
 */
public class RecognizeSpecifiedcountOfBarcodes {

	public static void main(String[] args) {

		// ExStart: RecognizeSpecifiedcountOfBarcodes

		try {

			// Instantiate Aspose Storage Cloud API SDK
			StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

			// Instantiate Aspose BarCode Cloud API SDK
			BarcodeApi barcodeApi = new BarcodeApi(Configuration.apiKey, Configuration.appSID, true);

			// Set Filename of image.
			String name = "sample-barcode.jpeg";

			// The barcode type.
			// If this parameter is empty, autodetection of all supported types
			// is used.
			String type = "";

			// Set mode for checksum validation during recognition
			String checksumValidation = "On";

			// Set if FNC symbol stripping should be performed.
			Boolean stripFnc = Boolean.TRUE;

			// Sets recognition of rotated barcode
			Integer rotationAngle = Integer.MIN_VALUE;

			// Sets exact number of barcodes to recognize
			Integer barcodesCount = Integer.MIN_VALUE;

			// Set recognition of barcode inside specified Rectangle region
			Integer rectX = Integer.SIZE;
			Integer rectY = Integer.SIZE;
			Integer rectWidth = Integer.MIN_VALUE;
			Integer rectHeight = Integer.MIN_VALUE;

			// Set 3rd party cloud storage server (if any)
			String storage = "";

			// Set folder location at cloud storage
			String folder = "";

			// upload files to aspose cloud storage
			storageApi.PutCreate(name, "", "",
					new File(RecognizeSpecifiedcountOfBarcodes.class
							.getResource("/" + name).toURI()));

			// invoke Aspose.BarCode Cloud SDK API to read barcode
			BarcodeResponseList apiResponse = barcodeApi.GetBarcodeRecognize(
					name, type, checksumValidation, stripFnc, rotationAngle,
					barcodesCount, rectX, rectY, rectWidth, rectHeight,
					storage, folder);

			if (apiResponse != null && apiResponse.getStatus().equals("OK")) {
				List<Barcode> barcodes = apiResponse.getBarcodes();
				for (Barcode barcode : barcodes) {
					Log.d("apose-example","Codetext: " + barcode.getBarcodeValue()
							+ "\nType: " + barcode.getBarcodeType());
				}
			}

		} catch (Exception e) {
			e.printStackTrace();
		}
		//ExEnd: RecognizeSpecifiedcountOfBarcodes
	}

}
